decidim-admin/app/helpers/decidim/admin/newsletters_helper.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    # This module includes helpers to manage newsletters in admin layout
+    module NewslettersHelper
+      def participatory_spaces_for_select(form_object)
+        html = "<div class='grid-x grid-padding-x'>"
```

Comment:
Have you tried using `content_tag`?

Reply:
I tried it, but I didn't get it to render correctly. I'll try again.